### PR TITLE
feat: update symfony dependencies

### PR DESCRIPTION
**Pull request by maikschneider**
Tested with newer version of symfony packages: works like a charm!

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,13 +18,13 @@
...
-		"symfony/finder": "^4.4 || ^5.4 || ^6.2",
-		"symfony/filesystem": "^4.4 || ^5.4 || ^6.2",
-		"symfony/event-dispatcher": "^4.4 || ^5.4 || ^6.2",
+		"symfony/console": "^4.4 || ^5.4 || ^6.2 || ^7.0",
+		"symfony/dependency-injection": "^4.4 || ^5.4 || ^6.2 || ^7.0",
+		"symfony/config": "^4.4 || ^5.4 || ^6.2 || ^7.0",
+		"symfony/yaml": "^4.4 || ^5.4 || ^6.2 || ^7.0",
+		"symfony/finder": "^4.4 || ^5.4 || ^6.2 || ^7.0",
+		"symfony/filesystem": "^4.4 || ^5.4 || ^6.2 || ^7.0",
+		"symfony/event-dispatcher": "^4.4 || ^5.4 || ^6.2 || ^7.0",
```
[the remark this answers is not in the document]
Comment (maikschneider) [reply composer.json]: Good point, thanks. I adjusted the version constraint.

**Review by oliverklee — APPROVED** [review]

**Review by martin-helmich — CHANGES_REQUESTED** [review]
Thanks! :+1:  Could you also add Symfony 7 to the build matrix (`.github/workflows/php.yml`, so that this project is also tested against the v7 versions?

**Comment by maikschneider** [comment]
Yes, v7 is now added. I replaced the `6.2` and `6.3` with `6.4` as well.

**Review by oliverklee — APPROVED** [review]

**Review by martin-helmich — APPROVED** [review]
LGTM! 👍

**Review by oliverklee — APPROVED** [review]

**Review by martin-helmich — APPROVED** [review]
So, this looks good in general :+1:  -- 867a2bfa4307d9a9ab3644f8f3a3a6f1d425ee06 was a good catch, I didn't remember we had that workaround in there. Only reason the tests are still failing is because the upstream library `helmich/typo3-typoscript-parser` is not yet compatible with the symfony v7 libraries; I'll see if we can bump these as well...

**Comment by maikschneider** [comment]
Waiting for https://github.com/martin-helmich/typo3-typoscript-parser/pull/95 to be merged

**Pull request merged**